src/main/java/com/exactprosystems/remotehand/windows/actions/Open.java

```diff
@@ -54,13 +55,17 @@ public String run(WindowsDriverWrapper driverWrapper, Map<String, String> params
 		
 		String workDir = params.get("workdir");
 		String execFile = params.get("execfile");
+		String appArgs = params.get("appargs");
 
 		Path workDirPath = Paths.get(workDir).normalize();
 		Path exeFilePath = workDirPath.resolve(execFile).normalize();
 
 		DesiredCapabilities capabilities = driverWrapper.createCommonCapabilities();
 		capabilities.setCapability(MobileCapabilityType.APP, exeFilePath.toString().replace('/', '\\'));
 		capabilities.setCapability("appWorkingDir", workDirPath.toString().replace('/', '\\'));
+		if (StringUtils.isNotBlank(appArgs)) {
+			capabilities.setCapability("appArguments", appArgs);
```

Comment:
Are these changes expected in this commit? Looks like this is from different task.

Reply:
I will add this change to original task as requirements.